crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -1242,10 +1250,10 @@ def prepare_remote_nodes
           # find the controller in this cluster
           cluster_name = ServiceObject.cluster_name(cluster)
           cluster_env = "pacemaker-config-#{cluster_name}"
-          controller = ::Node.find(
-            "pacemaker_founder:true AND " \
-            "pacemaker_config_environment:#{cluster_env}"
-          ).first
+          founder_name = ::Node.find(
+            "pacemaker_founder:* AND pacemaker_config_environment:#{cluster_env}"
```

Comment:
Again, `pacemaker_founder:*` is not needed

Reply:
+1